src/mlpack/methods/ann/rbm/binary_rbm_policy_impl.hpp

```diff
@@ -0,0 +1,159 @@
+/**
+ * @file binary_rbm.hpp
+ * @author Kris Singh
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_RBM_BINARY_RBM_POLICY_IMPL_HPP
+#define MLPACK_METHODS_ANN_RBM_BINARY_RBM_POLICY_IMPL_HPP
+
+#include <mlpack/core.hpp>
+#include <mlpack/prereqs.hpp>
+
+#include "binary_rbm_policy.hpp"
+
+namespace mlpack {
+namespace ann {
+
+template<typename DataType>
+BinaryRBMPolicy<DataType>
```

Comment:
Correct if I am mistaken, but it seems according to the style guide we should write
`BinaryRBMPolicy<DataType>::BinaryRBMPolicy(`
without line-breaks.